run.sh

```diff
@@ -119,7 +164,16 @@ while true; do
 
   # Ping kubelet for liveness
   f=kubelet_ping
-  c=$(curl -m 5 http://127.0.0.1:10255/healthz 2>&1 || true)
+
+  set +e
+  if ! c=$(curl -m "5" -f -s -S http://127.0.0.1:10248/healthz 2>&1); then
+    # Print the response and/or errors.
+    if [[ "${currentFtraces}" -lt "${maxFtraces}" ]]; then
```

Comment:
I was thinking to keep this separate, basically have this script call out to a hook.sh if we detect kubelet having issues.

```
f=kubelet_ping
set +e
 if ! c=$(curl -m "5" -f -s -S http://127.0.0.1:10248/healthz 2>&1); then
  /hook.sh
fi
set -e
output

Reply:
Sounds good, moved to a separate file hook.sh, but I sourced the file (to load the `do_ftrace` function) instead of calling directly to make it easier to pass variables and such.